.github/workflows/linux_upstream_test_moab.yml

```
@@ -1,4 +1,4 @@
-name: Test against MOAB master on PR merge
+name: Test against newer MOAB on PR merge
```

I think the original version is more specific, but maybe to combine the two:

```suggestion
name: Test against MOAB development branch (master) on PR merge
```